tests/php/Authenticator/ChangePasswordHandlerTest.php

```diff
@@ -34,6 +36,12 @@ protected function setUp(): void
         Config::modify()
             ->set(MethodRegistry::class, 'methods', [Method::class])
             ->set(Member::class, 'auto_login_token_lifetime', 10);
+
+        foreach (['tempHashAlreadyGenerated', 'tempHashAlreadyProcessed'] as $property) {
+            $refl = new ReflectionProperty(SecurityChangePasswordHandler::class, $property);
+            $refl->setAccessible(true);
+            $refl->setValue(null, false);
+        }
```

Comment:
Should probably be in `tearDown()` instead

Reply:
Updated